client/build/css/style.css

```diff
@@ -0,0 +1,181 @@
+body{
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  line-height: 1.5;
+  padding: 0;
+  margin: 0 auto;
+  background: #f4f4f4;
+  width: 98%;
+  margin-top: 30px
+}
+
+
```

Comment:
Keep spacing between classes/ids consistent. Also add a space between the class name declaration and the brace.

`.class {`